spec/json_matchers/match_response_schema_spec.rb

```diff
@@ -87,30 +87,67 @@
 
   it "does not fail when the schema matches" do
     create_schema("array_schema", {
-      "type" => "array",
-      "items" => { "type" => "string" },
+      "type": "array",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.